src/components/communities/BannedUsersPage.tsx

```diff
@@ -0,0 +1,133 @@
+import React, { useEffect, useState } from "react";
+import { Alert, StyleSheet } from "react-native";
+import { FlatList } from "react-native-gesture-handler";
+
+import {
+  BannedUsersPageNavigationProp,
+  BannedUsersPageRouteProp,
+} from "../../routes";
+import Box from "../themed/Box";
+import ThemedIcon from "../themed/ThemedIcon";
+import ThemedListItem from "../themed/ThemedListItem";
+import ThemedRefreshControl from "../themed/ThemedRefreshControl";
+
+// React Navigation Types and Page Options
+
+type BannedUsersPageProps = {
+  navigation: BannedUsersPageNavigationProp;
+  route: BannedUsersPageRouteProp;
+};
+
+export const BannedUsersPageOptions = {
+  title: "Banned Users",
+};
+
+// Page Styles
+
+const styles = StyleSheet.create({
+  root: {
+    flex: 1,
+    flexDirection: "column",
+    justifyContent: "flex-start",
+  },
+});
+
+// Dummy user info used for now
+type User = {
+  id: number;
+  firstName: string;
+  lastName: string;
+  profileImage: string;
+};
+
+const showUnbanDialog = (
+  user: User,
+  onConfirm: (user: User) => void,
+  onCancel?: () => void
+) => {
+  const { firstName, lastName } = user;
+  Alert.alert(
+    `Unban ${firstName} ${lastName}?`,
+    `Are you sure you want to unban ${firstName} ${lastName} from this community?`,
+    [
+      {
+        text: "Cancel",
+        style: "cancel",
+        onPress: onCancel,
+      },
+      {
+        text: "OK",
+        onPress: () => onConfirm(user),
+      },
+    ]
+  );
+};
+
+// Page Definition
+
+export default function BannedUsersPage({}: BannedUsersPageProps) {
+  // Create state for refreshing data and list of banned users
+  const [isRefreshing, setIsRefreshing] = useState(false);
+  const [bannedUsers, setBannedUsers] = useState<User[]>([]);
+  // Create function to unban a user on the backend
+  // TODO: Integrate functionality with backend
+  const unbanUser = (user: User) => {
+    bannedUsers.splice(bannedUsers.indexOf(user), 1);
+    setBannedUsers([...bannedUsers]);
+  };
+  // Create function to load banned user data
+  // TODO: Inegrate functionality with backend
+  const loadBannedUserData = async () => {
+    setIsRefreshing(true);
+    // TODO: Fetch data from the backend to get updated users list
+    setTimeout(() => {
+      setBannedUsers([
+        ...bannedUsers,
+        {
+          id: Math.random() * 10000000,
+          firstName: "John",
+          lastName: "Smith" + bannedUsers.length,
+          profileImage:
+            "https://t4.ftcdn.net/jpg/02/15/84/43/240_F_215844325_ttX9YiIIyeaR7Ne6EaLLjMAmy4GvPC69.jpg",
+        },
```

Comment:
if this is for testing can we remove it?

Reply:
Yes we can. Do we currently have a model setup for account info? I know we have the azure info but is that the same as our account?

Reply:
yes since my PR was merged in there should be something in `src/models/accounts.ts`

Reply:
Okay, the only key piece of data missing was the profile image url. So I didn't know if that was used specifically for azure

Reply:
you can add profile image to the model as an optional parameter `profileImage?: string;` so it doesnt break anything

Reply:
Okay, I'll go ahead an update this page to use that model and make the optional change to it. It may be helpful to also include a UID on the model if that is something we can get from the backend. I'll also remove the testing code here

Reply:
sounds good and I can ask about the UID